src/openrct2/ride/ride.h

```diff
@@ -219,9 +219,9 @@ typedef struct rct_ride {
 	union {
 		uint8 inversions;			// 0x114 (???X XXXX)
 		uint8 holes;				// 0x114 (???X XXXX)
-		// The undercover portion is a very rough approximation of how much of the ride is undercover.
+		// This a very rough approximation of how much of the ride is undercover.
```

Comment:
Your auto replacement seems to have just deleted " undercover portion " instead of replacing it with " sheltered eighths ".  If you meant it to be a this after all, you need to add an is afterwords.